domains/killuazoldyck.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "killuazoldyckreal",
+        "email": "bk1030798@gmail.com"
+    },
+    "record": {
+        "CNAME": "killuazoldyck.is-a.dev"
```

Comment:
Invalid CNAME record
CNAME records cant be the same domain

Reply:
corrected